### PR TITLE
added bulgarian language in settings menu, updated bulgarian language settings and fixed build for macOS

### PR DESCRIPTION
**Pull request by RosenDev**

### CODE REVIEW

**Comment by RosenDev** [comment]
@jarradizzle can you take a look please? Thanks!

**Comment by RosenDev** [comment]
@jostrander can you take a look please? Thanks!

File: src/_locales/bg-BG.json [hunk 1]
```diff
@@ -64,16 +64,16 @@
...
+  "playback-label-previous-track": "Предишен трак",
   "playback-label-stop": "Спри",
   "playback-label-thumbs-down": "Палец нагоре",
   "playback-label-thumbs-up": "Палец надолу",
   "playback-label-volume-down": "Намали звука",
   "playback-label-volume-up": "Увеличи звука",
-  "playback-label-info-track": "Уведоми ме за текущия track",
-  "playback-os-no-track-playing": "Не се изпълнява Track",
+  "playback-label-info-track": "Уведоми ме за текущия трак",
+  "playback-os-no-track-playing": "Не се изпълнява трк",
```
Comment (jostrander) [inline src/_locales/bg-BG.json:76]: Typo? Should be трaк?

**Comment by jostrander** [comment]
I should clarify my change request. Please keep pull requests scoped to a single issue. If there's an issue with macOS builds it should be a separate issue and PR.

File: src/_locales/bg-BG.json (hunk 1 above)
Comment (RosenDev) [reply src/_locales/bg-BG.json]: Yes, thanks. Fixed it.

**Comment by RosenDev** [comment]
> I should clarify my change request. Please keep pull requests scoped to a single issue. If there's an issue with macOS builds it should be a separate issue and PR.

Yes ok, thanks for letting me know.

**Comment by RosenDev** [comment]
@jostrander I applied the requested changes and added missing translate from en-US in bg-BG to fix failing linux test.

**Pull request merged**